mantidproject/mantid
# Normalise by maximum in Superplot

**Description of work.**

This PR adds a "normalise by maximum value" option in the Superplot. This feature was requested by our users.

It also adds the possibility to resize the left menu. This is particularly useful when a workspace with a really long name is present in the ADS.

**To test:**

* play with the checkbox to toggle the plot normalisation
* create a workspace with a really long name and check that the size of the left menu of the superplot can be changed

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
